azurecompute/src/main/java/org/jclouds/azurecompute/compute/AzureComputeServiceAdapter.java

```diff
@@ -88,17 +339,70 @@ public void resumeNode(String id) {
    @Override
    public void suspendNode(String id) {
       // TODO Auto-generated method stub
-
```

Comment:
Is this intentionally unimplemented? Should it be better to throw an exception if this is not supported?